src/bridge/views/SentTransaction.js

```diff
@@ -91,8 +91,8 @@ class Success extends React.Component {
 
           <H3>{ 'Transaction Status' }</H3>
           <P>
-            <div>{ status }</div>
-            <div>{ esanchor }</div>
+            { status }
+            { esanchor }
```

Comment:
I think you just want to give these their own separate `<P>` tags.
<img width="422" alt="image" src="https://user-images.githubusercontent.com/3829764/56233207-e07fd600-6082-11e9-9342-046cacdd38d8.png">

Reply:
Oh shit! Fixed.

Reply:
Actually, one more fix needed.

Reply:
This looks good to me rn, but let me know when you're truly done.

Reply:
Allright we're good now, made 3 small commits which should explain themselves.